app/services/user_merger.rb

```diff
@@ -172,6 +173,29 @@ def merge_user_visits
     DB.exec(sql, source_user_id: @source_user.id, target_user_id: @target_user.id)
   end
 
+  def merge_bookmarks
+    if @acting_user
+      ::MessageBus.publish "/merge_user",
+                           { message: I18n.t("admin.user.merge_user.merging_bookmarks") },
+                           user_ids: [@acting_user.id]
```

Comment:
I think it'll be useful to have test coverage for this too to ensure that we are publishing the right message to the right channel and user.

https://github.com/discourse/discourse/blob/c4f4130027b536daf06b1b64cb05c4c6bbff6a48/spec/lib/post_action_destroyer_spec.rb#L22-L29